packages/chrome/src/styled/sheet/StyledSheetWrapper.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { ThemeProps, DefaultTheme } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+import { SHEET_SIZE } from './StyledSheet';
+
+const COMPONENT_ID = 'chrome.sheet_wrapper';
+
+interface IStyledSheetWrapperProps {
+  size?: number;
+}
+
+export const StyledSheetWrapper = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledSheetWrapperProps & ThemeProps<DefaultTheme>>`
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  transition: opacity 0.5s ease-in-out;
```

Comment:
I _really_ think this transition needs to be removed. It does not jive with the Garden system overall. In the case of modals, the UX is presenting (what appears to the user) _new_ content. As such, a fade transition makes sense there.

In the case of this and `Accordion` etc. the UX is presenting (again, what appears to be) _existing_ content. The UI should appear as if the information was there all along. Not as if it is somehow being summoned into the view.

Reply:
Agreed, will remove this.